Makefile

```diff
@@ -94,7 +95,7 @@ checkstyle: test-shellcheck test-yaml
 	PERL5LIB=lib/perlcritic:$$PERL5LIB perlcritic lib
 
 .PHONY: test
-ifeq ($(TRAVIS),true)
+ifeq ($(CI),true)
```

Comment:
run-tests-within-container is currently applicable only to Travis: CircleCI does execute `make test` identically no normal non-containerized environment.

Reply:
I know. I am thinking about removing this part completely